src/js/tooltip.ts

```diff
@@ -20,6 +20,7 @@ export function reposition(e: MouseEvent, messageOverride?: string) :void {
     x = x + gap;
   }
   y = y - tip.clientHeight - gap;
+  if (y < 0) y = 0;
```

Comment:
This works in the isolated case of the top right corner, however, at some point in the future:

* the toolbar position may become optional (left | right | top | bottom), requiring both x and y positions  to be managed.
* notes may be added to links that would appear as tooltips.

We dont' have issues suggesting these things yet, but for these reasons orienting towards the screen centre woudl be _more_ useful!